### ISSUE TITLE
Inline confirm for pull from postmaster

### ISSUE THREAD

**Issue by bhollis** [DestinyItemManager/DIM#7215]
The scourge of accidentally hitting the pull from postmaster button continues to ruin carefully placed item arrangements to the dismay of some users. We should have an inline confirm, changing this to effectively require a double tap. The inline confirm may be useful for AWA too.

**Comment by bhollis** [comment]
Alternately I guess we could just have a setting in the settings page to remove the button entirely.

**Comment by chabala** [comment]
Would like a feature flag in user settings for #7824, I don't want/need a confirmation click for the postmaster.

**Comment by bhollis** [comment]
We generally don't add user settings. If you have an issue with this please elaborate what problem you are having with the confirm.

**Comment by chabala** [comment]
I find it an annoyance, enough that I've located the PR that introduced it and the initial issue and come here to comment. I've never accidentally pulled from the postmaster, nor does my gameplay style involve treating the postmaster as extra vault space that needs careful management. For me, this change just adds another click, as though clicking it once wasn't good enough.

I'd consider a greasemonkey script to remove the change made here, but that's fragile when it's better managed in-project with a user setting.

**Comment by bhollis** [comment]
OK - can you wait a week, and then come back and let us know if it's still super annoying after you've had some time to get used to the new muscle memory?

**Comment by nneitzke** [comment]
Hate to be negative because I love DIM, but I also find this tedious and annoying.  In addition doesn't always seem to register the click so it feels like I have to hit the button 10 times to get it to register.  A setting would be very appreciated.

**Comment by samvbeckmann** [comment]
Yeah, used the new mode for a week, and went to find the user setting to turn it off, turns out there isn't one.

**Comment by chabala** [comment]
I'm still not a fan of the confirmation. I find myself missing the second click, or having to watch and see if the second click registers, and a lot of triple clicking just to be sure.

DIM as a whole is invaluable, and I appreciate it, and appreciate Destiny exposing APIs to make it possible.

**Comment by nev-r** [comment]
that's fine and fair. gonna make this an opt-in setting. thanks for trying it out and testing usability

**Comment by mavsss** [comment]
thank you! I also find the confirmation unnecessary.
